web/themes/joinup/templates/form/container--grid.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
```

Comment:
Original container.html.twig is in /core/themes/stable/templates/form, usually I place templates in the same folder as it is in the base theme.